pkg/config/database.go

```diff
@@ -31,9 +38,33 @@ func (d *Database) Open(logger *zap.SugaredLogger) (*icingadb.DB, error) {
 		driver.Register(logger)
 	})
 
-	dsn := fmt.Sprintf(
-		"%s:%s@tcp(%s:%d)/%s?timeout=60s",
-		d.User, d.Password, d.Host, d.Port, d.Database)
+	myCfg := mysql.NewConfig()
+
+	myCfg.User = d.User
+	myCfg.Passwd = d.Password
+	myCfg.Net = "tcp"
+	myCfg.Addr = net.JoinHostPort(d.Host, fmt.Sprint(d.Port))
```

Comment:
Nice! That change should also fix an issue with IPv6 address literals in the host parameter :heart: